### ISSUE TITLE
Regression

### ISSUE THREAD

**Issue by ronan-mch** [ronan-mch/machine_learning#5]
- [x] Explain which regression problem you have chosen to solve.
- [x] Apply linear regression with forward selection and consider if transforming or combining attributes potentially may be useful. For linear regression, plotting the residual error vs. the attributes can give some insight into whether including a transformation of a variable can improve the model, i.e. potentially describe parts of the residuals.
- [x] Explain how a new data observation is predicted according to the estimated model. I.e. what are the effects of the selected attributes in terms of predicting the data. _(Notice, if you interpret the magnitude of the estimated coefficients this in general requires that each attribute be normalized prior to the analysis.)_.
- [x] Fit an artificial neural network (ANN) model to the data.
- [x] Statistically evaluate if there is a significant performance difference between the fitted ANN and linear regression models based on the same cross-validation splits (i.e., use a paired t-test). Compare in addition if the performance of your models are better than simply predicting the output to be the average of the training data output.
